dbt/models/indexed/contrats_sorties.sql

```diff
@@ -29,10 +29,22 @@ select distinct
         to_date(ctr.contrat_date_sortie_definitive, 'DD/MM/YYYY')
     ) || '-01-01'))                                                                 as debut_annee_fin_contrat,
     /* 0.0328767 = the value to convert days to months */
-    (rcrt.date_sortie_definitive::DATE - rcrt.date_recrutement::DATE) * (0.0328767) as duree_en_mois
+    (rcrt.date_sortie_definitive::DATE - rcrt.date_recrutement::DATE) * (0.0328767) as duree_en_mois,
+    case
+        when ctr.contrat_salarie_rsa = 'OUI-M' then 'OUI'
+        when ctr.contrat_salarie_rsa = 'OUI-NM' then 'OUI'
```

Comment:
Pas testé, mais ça marcherais pas ça ?
```suggestion
        when ctr.contrat_salarie_rsa in ('OUI-M', 'OUI-NM') then 'OUI'
```
Ou éventuellement
```suggestion
        when ctr.contrat_salarie_rsa ^@ 'OUI-' then 'OUI'  -- ^@ == starts_with()
```